Bugzilla/WebService/Product.pm

```diff
@@ -227,11 +229,11 @@ sub _component_to_hash {
         $field_data->{flag_types} = {
             bug =>
                 [map {
-                    $self->_flag_type_to_hash($_)
```

Comment:
Crazy that we never even passed a $params to $self->_flag_to_hash() so the filter() call was a waste of time :)